src/actionsProvider.ts

```diff
@@ -40,56 +34,54 @@ export class ActionsProvider {
 	] //An advanced callback will be applied to this commands.
 
 	private drModuleInstance: DRModuleInstance
-	drActionInfos: DrActionInfo[]
 
 	constructor(drModuleInstance: DRModuleInstance) {
 		this.drModuleInstance = drModuleInstance
-		this.drActionInfos = []
 	}
 
-	getActions(): CompanionActions {
+	getActions(): CompanionActionDefinitions {
 		return {
 			[ActionNames.windowFullscreen]: {
-				label: CompanionLabels.windowFullScreen,
+				name: CompanionLabels.windowFullScreen,
 				options: [],
 				subscribe: (action: CompanionActionEvent) => {
 					this.subscribeAction(action, DRCommands.windowFullscreen)
 				},
 				unsubscribe: (action: CompanionActionEvent) => {
 					this.unsubscribeAction(action)
 				},
-				callback: (action, bank) => {
-					this.handleActionCallback(action, bank)
+				callback: async (action, bank) => {
```

Comment:
the parameter `bank` is deprecated and will be removed from the API.
You don't use it anyway, so please remove it.
Also applies to other callbacks.